### ISSUE TITLE
Problem with spaces in project name

### ISSUE THREAD

**Issue by marcelbischoff** [cedarbaum/fugitive-azure-devops.vim#5]
Space in project name becomes `%2520` instead of `%20`

**Comment by cedarbaum** [comment]
Really sorry about the late reply on this, I missed this issue unfortunately. This was previously reported in https://github.com/cedarbaum/fugitive-azure-devops.vim/issues/4 but I was unable to reproduce it at first.

I took another look and have now been able to reproduce the issue. My current theory is that this is a bug with how fugitive passes the file path to this plugin. I've opened an issue to get the maintainer's thoughts (https://github.com/tpope/vim-fugitive/issues/2230), so will see if the issue is indeed there or if I need to apply a fix to this project.
